SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-private-mdc-mapping.adoc

```diff
@@ -23,7 +23,7 @@ The following example uses the SDC *pm:Type* to demonstrate the mapping of priva
 |CWE-2
 |Text
 |"*MDC_PRIVATE_*<CWE-1>"
-|Since this is a private code, a MDC RefId does not exist for the code. The field shall be set to a private RefId which consists of the prefix *"MDC_PRIVATE_"* plus the private code set in CWE-1.
+|Since this is a private code, a MDC RefId does not exist for the code. The field is to be set to a private RefId that consists of the prefix *"MDC_PRIVATE_"* plus the private code set in CWE-1.

```

Comment:
Question: Is it possible that a private code starts with MDCX_? (Maybe in cases where it is private but intended to be standardized.) If so, we should add this. Proposal: [...] that consists of the prefix MDC_PRIVATE_ or MDCX_ plus [...]

Reply:
@MartinKasp: my understanding is that MDCX_... RefIds / codes are managed by the 11073-10101 / Rosetta workgroup. MDCX_... terms are not considered private codes in this context and can be used since they should be unique and part of the MDC coding system. This section deals mainly with the private codes from the private code range in the 10101 nomenclature.

Thanks for the corrections above.

Reply:
Further discussion can be found in #337